docs/en/ingest-management/elastic-agent/configuration/elastic-agent-configuration.asciidoc

```diff
@@ -21,6 +21,32 @@ changes after installation, you must modify the installed file.
 
 For installation details, refer to <<install-standalone-elastic-agent>>.
 
+Alternatively, you can put input configurations in YAML files into the
+folder `{path.config}/inputs.d` to separate your configuration into
+multiple smaller files.
+Glob that has to match the path is `{path.config}/inputs.d/*.yml`.
+From external configuration files ONLY input configurations are loaded.
```

Comment:
I think the word order here makes this sentence a bit harder to parse. 

Maybe say something like this instead: "The YAML files in the `inputs.d` folder should contain input configurations only. Any other configurations are ignored."